src/main/resources/view/DarkTheme.css

```diff
@@ -316,3 +316,22 @@
     -fx-background-radius: 2;
     -fx-font-size: 11;
 }
+
+.tag_label {
+    -fx-text-fill: white;
+    -fx-padding: 1 3 1 3;
+    -fx-border-radius: 2;
+    -fx-background-radius: 2;
+    -fx-font-size: 11;
+}
+
+.details_title {
+    -fx-font-weight: bold;
+    -fx-font-size: 32;
```

Comment:
generally can decrease font size

also would be good to add a scrollbar to indicate that the right panel is scrollable ( can follow some of the fxml / css used for the result display)

Reply:
more importantly, when you change the window size there are some spaces that appear in the UI